src/app/types/HTTPError.ts

```diff
@@ -60,6 +60,7 @@ export enum HTTPError {
 
   CANNOT_ACQUIRE_LOCK = 510,
   INVALID_FILE_FORMAT = 583,
+  BAD_REQUEST = 400,
```

Comment:
No need this BAD_REQUEST.
More over don't confuse HttpError with HttpAuthError.

Reply:
deleted and modified error handling to forbidden